glean_parser/schemas/pings.1-0-0.schema.yaml

```diff
@@ -54,6 +54,13 @@ additionalProperties:
         When `true`, include the `client_id` value in the ping.
       type: boolean
 
+    send_if_empty:
+      title: Send if empty
+      description: |
+        When `false` (the default) a ping is send only if it contains data.
```

Comment:
```suggestion
        When `false` (the default) a ping is sent only if it contains data.
```

Reply:
Note: I might be wrong, but it sounded weird. Maybe a @chutten check?

Reply:
"a ping is sent" is correct if you are asking about English grammar